src/sentry/static/sentry/app/views/performance/landing/chart/durationChart.tsx

```diff
@@ -7,6 +7,7 @@ import {Location} from 'history';
 import {Client} from 'app/api';
 import ErrorPanel from 'app/components/charts/errorPanel';
 import EventsRequest from 'app/components/charts/eventsRequest';
+import {HeaderTitleLegend} from 'app/components/charts/styles';
```

Comment:
when looking at this absolute path, I would expect to get only css styles. wouldn't it be better to create separate components for the ones we need to import?

Reply:
We have a few modules like this, perhaps we can come up with a better module name for these in a separate set of changes.